Source/Evolve/WarpXComputeDt.cpp

```diff
@@ -33,8 +33,12 @@ void
 WarpX::ComputeDt ()
 {
     // Handle cases where the timestep is not limited by the speed of light
-    if (electromagnetic_solver_id == ElectromagneticSolverAlgo::None) {
-        WARPX_ALWAYS_ASSERT_WITH_MESSAGE(m_const_dt.has_value(), "warpx.const_dt must be specified with the electrostatic solver.");
+    if (electromagnetic_solver_id == ElectromagneticSolverAlgo::None ||
```

Comment:
Can this be separated out into the different cases so that the assert error message is specific to the setup?

Reply:
Yeap, can do!